cpp/openlocationcode.cc

```diff
@@ -199,7 +199,7 @@ std::string Encode(const LatLng &location) {
 
 CodeArea Decode(const std::string &code) {
   // Make a copy that doesn't have the separator and stops at the first padding
-  // character.
+  // character, and is constrained to the maximum length.
   std::string clean_code(code);
   clean_code.erase(
```

Comment:
The code from lines 203-210 seems to be duplicated in CodeLength(). Can you extract and de-dupe?

Reply:
Sure thing! Done.